packages/beacon-node/src/chain/blocks/verifyBlock.ts

```diff
@@ -174,6 +174,17 @@ export async function verifyBlocksInEpoch(
           default:
         }
       }
+
+      if (isForkPostFulu(fromForkBoundary.fork)) {
+        const fromBlobParameters = this.config.getBlobParameters(fromForkBoundary.epoch);
+        const toBlobParameters = this.config.getBlobParameters(toForkBoundary.epoch);
+
+        if (toBlobParameters.maxBlobsPerBlock !== fromBlobParameters.maxBlobsPerBlock) {
```

Comment:
```suggestion
        if (toBlobParameters.epoch !== fromBlobParameters.epoch) {
```
I think you should compare `BlobParameters.epoch` here. Since blob schedule only enforces `EPOCH` to be unique, but not `MAX_BLOBS_PER_BLOCK`

Reply:
> we should probably enforce first bpo entry's epoch to be greater or equal to FULU_FORK_EPOCH in validateBlobSchedule()

yeah can do that although we have blob parameters fork guarded and not use it pre-fulu, it would just return that pre-fulu epoch instead of default electra values but it's good to catch misconfiguration

Reply:
> edit: we should probably enforce first bpo entry's epoch to be greater or equal to FULU_FORK_EPOCH in validateBlobSchedule()

so the problem is that we don't have access to config in `validateBlobSchedule` as we just parse the config at this point